src/lib/utils/thorchain/memo/assertAndProcessMemo.ts

```diff
@@ -82,13 +82,33 @@ export const assertAndProcessMemo = (memo: string): string => {
     case '=':
     case 's': {
       // SWAP:ASSET:DESTADDR:LIM/INTERVAL/QUANTITY:AFFILIATE:FEE
-      const [_action, asset, destAddr, limit, , fee] = memo.split(':')
+      const [
+        _action,
+        asset,
+        destAddr,
+        limit,
+        ,
+        fee,
+        aggregatorLastTwoChars,
+        finalAssetAddressShortened,
+        finalAssetAmountOut,
+      ] = memo.split(':')
 
       assertMemoHasAsset(asset, memo)
       assertMemoHasDestAddr(destAddr, memo)
       assertIsValidLimit(limit, memo)
 
-      return `${_action}:${asset}:${destAddr}:${limit}:${THORCHAIN_AFFILIATE_NAME}:${fee || 0}`
+      const swapOutParts = (() => {
```

Comment:
```suggestion
      const maybeSwapOutParts = (() => {
```

Reply:
And add the same as l.84 for reference on top of this guy:

      // SWAP:ASSET:DESTADDR:LIM:AFFILIATE:FEE:DEXAggregatorAddr:FinalTokenAddr:MinAmountOut|
      
https://dev.thorchain.org/aggregators/memos.html

Reply:
Done in 03978b18e16